app/controllers/attended_events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class AttendedEventsController < ApplicationController
+  before_action :logged_in_user, only: [:create]
+  before_action :check_subscription_status, only: [:create]
+
+  
+  def create
+    @user = current_user().id
```

Comment:
an instance variable isn't needed here, if you are taking the id from current user method, pass it as such to attendee_id.

Reply:
changed that line to... thanks

`attendee_id = current_user().id`